ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -972,3 +972,60 @@ def test_numpy_triangular(
         right=right,
         size=size,
     )
+
+
+# noncentral_f - numpy
+@handle_frontend_test(
+    fn_tree="numpy.random.noncentral_f",
+    input_dtypes=helpers.get_dtypes("float"),
+    dfnum=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=1,
+        max_value=1000,
+        exclude_min=True,
+    ),
+    dfden=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=1,
+        max_value=1000,
+        exclude_min=True,
+    ),
+    nonc=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=0,
+        max_value=1000,
+    ),
+    size=helpers.get_shape(allow_none=False),
+)
+def test_numpy_noncentral_f(
+    input_dtypes,
+    size,
+    frontend,
+    test_flags,
+    backend_fw,
+    fn_tree,
+    on_device,
+    dfnum,
+    dfden,
+    nonc,
+):
+    test_flags.num_positional_args = 3
```

Comment:
This shouldn't be needed. The testing piepline should be able to detect this automatically.

Reply:
@hello-fri-end,

I have removed this.

Looking forward to your feedback.

Regards,